src/routes/api.js

```diff
@@ -1,10 +1,15 @@
 const express = require("express");
 const router = express.Router();
 const { businesses, businessesId, newReview } = require("../controllers/api");
+const { aw3 } = require("../controllers/aw3");
```

Comment:
give it a name better than `aw3` it's not a very clear name

Reply:
fixed